src/com/bjorn/MakeChange/IMakeChange.java

```diff
@@ -0,0 +1,12 @@
+package com.bjorn.MakeChange;
+
+import java.util.HashMap;
+
+/**
+ * Created by bjornjohnson on 1/6/16.
+ */
+public interface IMakeChange {
+    void updateCoinAmountHashMap(String cointType, HashMap<String, Integer> hmap);
```

Comment:
Also, encoding type information in names is a smell in typed languages. I should know that this updates a hashmap based on the name and the parameters I'm passing. What is this really doing and how could it be better named?